src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from "react";
+import { NavLink, useNavigate } from "react-router-dom";
+import { Button, Form, Container, Card } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as Yup from "yup";
+import ky from "ky";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import "../styles/Auth.scss";
+import { useDispatch } from "react-redux";
+import { loginUser } from "../store/actionCreators";
+
+type UserLoginForm = {
+  email: string;
+  password: string;
+};
+
+type ResponseLogin = {
+  token: string;
+  message: string;
+};
+
+const LoginPage: FC = () => {
```

Comment:
This class handles a lot of logic(more than one responsibility) and required the refactoring

Reply:
Also, I recommend you to avoid the React.FC usage https://github.com/facebook/create-react-app/pull/8177